pinot-controller/src/main/java/com/linkedin/pinot/controller/ControllerStarter.java

```diff
@@ -161,13 +161,15 @@ public void start() {
     _helixResourceManager.start();
     final HelixManager helixManager = _helixResourceManager.getHelixZkManager();
 
+    LOGGER.info("Creating ControllerLeadershipManager instance");
+    ControllerLeadershipManager.createInstance(helixManager);
```

Comment:
I put it at this place because helixManager is initialized in pinotHelixResourceManager.start(). If we were to move it before the start, we would have to extract out the helixManager setup part